lib/awful/titlebar.lua

```diff
@@ -754,14 +811,20 @@ end
 -- @constructorfct awful.titlebar.widget.button
 function titlebar.widget.button(c, name, selector, action)
     local ret = imagebox()
-
     if titlebar.enable_tooltip then
-        ret._private.tooltip = atooltip({ objects = {ret}, delay_show = 1 })
-        ret._private.tooltip:set_text(name)
+        ret._private.tooltip = atooltip({
+            objects = {ret},
+            delay_show = beautiful["titlebar_tooltip_delay_show"] or 1.0,
```

Comment:
This is little bit driven by more advanced library "Nice", where is much more better driven placement and timing of tooltip. If you will test the tooltips longer time, you will se that there is problem of clicking to buttons, because tooltip is shown across the button. You have to click to tooltip first, tooltip will be closed, and after that you have chance to click to button. This happens very often. The "nice" plugin solves this problem with better placement of tooltip to the right corner with timing. There is no double click required , you just correct click to button without closing of tooltip. 
It is main reason why I added these configuration parameters, because standard behaviour of default theme is not "correct". So, I tried to open possibility to configure better behaviour which is known in "nice" component. The standard default settings is still kept.

Reply:
I understand why you made these changes, and I'm happy to see more configuration available here to allow the user to better place the tooltip.

My comment was an open question to understand why you had to change the value for `delay_show` from `1` to `1.0`. Whether there is any technical aspect, I'm not aware of.

Reply:
Reverted to 1.